mitosheet/mitosheet/tests/step_performers/test_export_to_file.py

```diff
@@ -86,6 +87,54 @@ def get_cell_conditional_formatting(
     "conditional_formats": []
 }
 
+DF_NUMBER_FORMATS = [
+    (
+        {
+            "A": {
+                "type": CURRENCY,
+                "precision": 3
+            }
+        },
+        "$0.000"
+    ),
+    (
+        {
+            "A": {
+                "type": SCIENTIFIC_NOTATION,
+                "precision": 1
+            }
+        },
+        "0.0E+0"
+    ),
+    (
+        {
+            "A": {
+                "type": PLAIN_TEXT,
+                "precision": 0
+            }
+        },
+        "0"
+    ),
+    (
+        {
+            "A": {
+                "type": PERCENTAGE,
+                "precision": 2
+            }
+        },
+        "0.00%"
+    ),
+    (
+        {
+            "A": {
+                "type": ACCOUNTING,
+                "precision": 1
+            }
+        },
+        "($0.0)"
+    )
+]
+
```

Comment:
Here's two tests that aim to flex this new feature to find some bugs:
1. Two columns with number formats. 
2. Formatting with number formats and conditional formats.